src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,22 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null) {
+            throw new RuntimeException("Login can`t be null");
+        }
+        if (storageDao.get(user.getLogin()) != null) {
+            throw new RuntimeException("User with such login already exists");
+        }
+        if (user.getAge() < 1) {
+            throw new RuntimeException("Impossible age");
+        }
+        if (user.getAge() < 18) {
+            throw new RuntimeException("Oh my boi, U R 2 young 4 dat");
```

Comment:
You should create your custom unchecked exception and throw it in case of invalid data.
Use more informative messages